check_api/src/main/java/com/google/errorprone/util/FindIdentifiers.java

```diff
@@ -153,7 +153,7 @@ public static Symbol findIdent(String name, VisitorState state, KindSelector kin
           com.sun.tools.javac.util.List<Type> superTypes = state.getTypes().closure(classType).tail;
           for (Type type : superTypes) {
             Scope scope = type.tsym.members();
-            ImmutableList.Builder<VarSymbol> varsList = new ImmutableList.Builder<VarSymbol>();
+            ImmutableList.Builder<VarSymbol> varsList = new ImmutableList.Builder<>();
```

Comment:
Good catch!

I've just pushed a new commit with `new Immutable.Builder` usages replaced with `ImmutableList.builder()` for your reviewing pleasure.